cdci_data_analysis/analysis/parameters.py

```diff
@@ -476,14 +457,11 @@ def __init__(self, value=None, T_format='isot', name=None, Time_format_name=None
 
         self._set_time(value, format=T_format)
 
-    def set_par(self, value, units=None):
-        # sets the value to the format initially specified in the form
-        if units is not None:
-            self.units = units
-        self.value = value
-        # return the relative isot format value
-        # afterwards also the units_name will be set to isot
-        return self._astropy_time.isot
+    def get_value_default_format(self, value):
```

Comment:
this is the kind of case I would use type hint, at least to str. 
Not even for just static analysis, for for explaining to ourselves our intent

```suggestion
    def get_value_default_format(self, value) -> str:
```